src/index.js

```diff
@@ -7,5 +7,13 @@ document.querySelector('#add-form').addEventListener('submit', (e) => {
   // prevent default submit
   e.preventDefault();
 
-  UI.getNewScore();
-});
\ No newline at end of file
+  const name = document.querySelector('#name').value;
+  const score = document.querySelector('#score').value;
+
+  UI.getNewScore(name, score);
+  UI.deleteScore('John Doe', 42);
+  UI.deleteScore('farzad', 67);
+  UI.deleteScore('amir', 20);
+});
```

Comment:
- [OPTIONAL] Even though this piece of code is not affecting the functionality of the application, it gives error messages in the browser's console. 

![image](https://user-images.githubusercontent.com/77106962/233398145-603c0104-936e-4e96-8647-812bc8632d74.png)

#### Kindly delete this code. 👍 💻

Reply:
Hi @Gambit142 ,

Thanks for your time and for reviewing my code.  I deleted those codes.